samples/hello-world-expo/README.md

```diff
@@ -0,0 +1,20 @@
+# RXPHelloWorld
+
+This app works on React Native (iOS, Android) and web. Most of the app's code is contained in `App.tsx`.
+
+The commands in the instructions below assume you are in the root of this repo.
+
+### Initial Setup
+
+- Run `npm install`. This fetches the dependencies.
+
+### Building for Web
+
+- Run `npm run web-watch`. This compiles the TypeScript code and recompiles it whenever any files are changed.
+- Open `index.html` in your browser to view the result.
+
+### Building for React Native
+
+Install the [Expo](https://expo.io) app on your iOS or Android phone, and use the QR code in the terminal to open your app. Find the QR scanner on the Projects tab of the app. When you're ready to share your project with others (for example, by deploying to an app store)
+
+- Run `exp start`
```

Comment:
I try it and it says:
Error: Missing exp.json. See https://docs.expo.io/

probably something is missing?